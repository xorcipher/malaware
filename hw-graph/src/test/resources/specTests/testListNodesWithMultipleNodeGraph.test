# Tests that the getNodes method works properly for a graph with multiple nodes.

# Return all the nodes of a graph in String format.

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C

AddEdge graph A B edgeA

ListNodes graph