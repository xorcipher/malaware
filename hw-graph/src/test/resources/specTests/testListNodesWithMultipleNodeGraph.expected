# Tests that the getNodes method works properly for a graph with multiple nodes.

# Return all the nodes of a graph in String format.

created graph graph

added node A to graph
added node B to graph
added node C to graph

added edge edgeA from A to B in graph

graph graph contains: A B C
